### PR TITLE
bjshare: Also parse the international titles from the 24h search

### PR DESCRIPTION
**Pull request by christianfds**
Now both kinds of search's uses the "ParseTitle" method.

Signed-off-by: Christian Franchin dos Santos <christian@corp.kionux.com.br>

### CODE REVIEW

File: src/Jackett.Common/Indexers/BJShare.cs [hunk 1]
```diff
@@ -141,8 +141,9 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
 
         private static string InternationalTitle(string title)
         {
-            var match = Regex.Match(title, @".* \[(.*?)\]$");
-            return match.Success ? match.Groups[1].Value : title;
+            var match = Regex.Match(title, @".* \[(.*\/?)\]");
+            string[] results = match.Groups[1].Value.Split('/');
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/BJShare.cs:145]: This line can throw an exception, change it for:
```
return match.Success ? match.Groups[1].Value.Split('/')[0] : title;
```

**Review by ngosang — CHANGES_REQUESTED** [review]

**Comment by ngosang** [comment]
@christianfds are you going to finish this PR?

**Comment by christianfds** [comment]
> @christianfds are you going to finish this PR?

I've been busy and forgot about it, i made the change as requested.

**Comment by ngosang** [comment]
thank you!

**Pull request merged**